grails-app/views/report/showInventoryByLocationReport.gsp

```diff
@@ -35,8 +35,27 @@
                                                   data-placeholder=" "
                                                   activityCode="${org.pih.warehouse.core.ActivityCode.MANAGE_INVENTORY}"/>
                             </div>
+                            <label><warehouse:message code="category.label"/></label>
+                            <p>
+                                <g:selectCategory
+                                        id="category" multiple="true"
+                                        class="chzn-select-deselect filter"
+                                        data-placeholder="${g.message(code: 'category.selectCategory.label', default: 'Select a category')}"
+                                        name="category"
+                                        noSelection="['':'']"
+                                        value="${params?.list('category')}"
+                                />
+                            </p>
+                            <p>
+                                <label>
+                                    <g:checkBox name="includeCategoryChildren" value="${params?.includeCategoryChildren}" checked="${includeCategoryChildren}"/>
```

Comment:
the initial value could be just set to `includeCategoryChildren` and you can get rid of `checked`, as it inherits from `value`:
```gsp
 <g:checkBox name="includeCategoryChildren" value="${includeCategoryChildren}" />
```